doc/source/ray-contribute/development.rst

```diff
@@ -170,7 +254,7 @@ Define an environment variable BAZEL_PATH to full exe path (example:
 Environment variables that influence builds
 --------------------------------------------
 
-You can tweak the build with the following environment variables (when running ``setup.py``):
+You can tweak the build with the following environment variables (when running ``setup.py`` or ``pip install -e .``):
```

Comment:
in contrast to the new command, maybe we should now say `python setup.py install`?

Reply:
Agreed! 🤓